### PR TITLE
Fetch Real Stuff From DB

### PR DESCRIPTION
**Pull request by harshkhandeparkar**
- [x] Fetch real channel details and display on channel page.
- [x] Fetch real messages and display.

### CODE REVIEW

File: excretor/src/models.rs [hunk 1]
```diff
@@ -2,9 +2,9 @@ use sqlx::prelude::FromRow;
 
 #[derive(FromRow)]
 pub struct Message {
-    pub id: i32,
     pub channel_name: String,
     pub user_id: String,
+    #[sqlx(rename = "msg_text")]
     pub text: String,
     #[sqlx(rename = "ts")]
     pub timestamp: String,
```
Comment (Devansh-bit) [inline excretor/src/models.rs:10]: This isnt a string, but an SQL timestamp apparently see https://stackoverflow.com/a/61670747 and https://stackoverflow.com/a/75904277

File: excretor/src/routing.rs [hunk 2]
```diff
@@ -30,8 +29,8 @@ mod handlers {
 
     #[derive(Deserialize)]
     pub struct Pagination {
-        page: usize,
-        per_page: usize,
+        last_msg_timestamp: Option<String>,
```
Comment (Devansh-bit) [inline excretor/src/routing.rs:32]: might work with previous timestamp conversion

**Review by Devansh-bit — APPROVED** [review]

File: excretor/src/routing.rs (hunk 2 above)
Comment (harshkhandeparkar) [reply excretor/src/routing.rs:33]: Done.

File: excretor/src/models.rs (hunk 1 above)
Comment (harshkhandeparkar) [reply excretor/src/models.rs]: Done.

**Review by chirag-ghosh — APPROVED** [review]
Looks quite rusty

**Pull request merged**